docs/src/pages/guides/auth-protected/auth-protected.react.mdx

```diff
@@ -0,0 +1,414 @@
+import { Image } from '@aws-amplify/ui-react';
+
+The [Amplify UI Authenticator](/components/authenticator) component allows you to quickly scaffold out an end-to-end authentication flow, in
+just a few lines of code.
+
+Typically, this can be accomplished by wrapping the entire application in the [withAuthenticator](components/authenticator?platform=react#quick-start) higher-order component.
+This forces the entire application behind a login page.
+
+Let's imagine you needed a web application where the login page was in it's own route, and that you needed a handful of selected routes
+to only be accessed by logged in users.
+
+This guide demonstrates that scenario while using the [Amplify UI Authenticator](/components/authenticator) and the [Amplify CLI](https://docs.amplify.aws/cli/).
+
+> If you'd like to follow along, please use this example [Github repo](https://github.com/ErikCH/protected-route-amplify-react). Additionally,
+> you can watch this [video](https://youtu.be/6f635LVtwgk) that walks through this process, and includes a bonus section on how to create a lambda function
+> with a custom authorizer.
+
+_Example Screenshot of App_
+
+<Image
+  alt="Example screenshot of the route guard application."
+  height={{ base: '25rem', medium: '25rem' }}
+  display="block"
+  objectFit="contain"
+  style={{ margin: '0 auto' }}
+  src="/example-auth-routes.png"
+/>
+
+### Setup
+
+To get started we'll need to create a new React application, by following these steps:
+
+```bash
+npx create-react-app new-auth-app
+cd new-auth-app
+```
+
+Next, install the Amplify libraries and React Router.
+
+```bash
+npm i @aws-amplify/ui-react aws-amplify react-router-dom@6
```

Comment:
Qq, why specifically `@6`?

Reply:
That's what the official guide shows. https://reactrouter.com/docs/en/v6/getting-started/installation#basic-installation

Reply:
7c6ee573f1e15b8407892dfd8552ea416e8cdf63